sysutils/helm/Portfile

```diff
@@ -183,7 +204,6 @@ if {${subport} == ${name}} {
 } else {
     PortGroup           github 1.0
 
-    supported_archs     x86_64
```

Comment:
There are other archs besides just `arm64` and `x86_64`, but per the if statement you have below, those are the only two this will work on as a result of this change.  Therefore `supported_archs` should list just those two.

```
supported_archs     arm64 x86_64
```